_content/booklets/the-discourse-on-right-view_nyanamoli.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "The Discourse on Right View: The Sammādiṭṭhi Sutta and its Commentary"
+authors:
+  - nyanamoli
+editor: bodhi
+external_url: "https://www.accesstoinsight.org/tipitaka/mn/mn.009.ntbb.html"
```

Comment:
On the BPS Wheel links you can just change `.pdf` to `.html` to get the html version: https://www.bps.lk/olib/wh/wh377_Nyanamoli_Discourse-on-Right-View--Sammaditthi-Sutta.html

Please use that link as the AtoI version just has the sutta and no commentary 😁